keps/sig-auth/4447-promote-policyreport-sig-api/README.md

```diff
@@ -0,0 +1,876 @@
+
+The Policy Report API is currently hosted at: https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report and is used by the following projects:
+
+**Report producers:**
+* [Kyverno](https://kyverno.io/docs/policy-reports/)
+* [jsPolicy](https://github.com/loft-sh/jspolicy/)
+* [Falco](https://github.com/falcosecurity/falcosidekick/blob/master/outputs/policyreport.go)
+* [Trivy Operator](https://aquasecurity.github.io/trivy-operator/v0.15.1/tutorials/integrations/policy-reporter/)
+* [Tracee Adapter](https://github.com/fjogeleit/tracee-polr-adapter)
+
+**Report consumers:**
+* [Kyverno Policy Reporter](https://kyverno.github.io/policy-reporter/)
+* [Open Cluster Management](https://open-cluster-management.io/)
+* [Lula](https://github.com/defenseunicorns/lula)
+* [Fairwinds Insights](https://fairwinds.com/insights)
+
+Additionally, the following tools were developed by the Policy WG:
+* [kube-bench-adapter](https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report/kube-bench-adapter)
+* [kubearmor-adapter](https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report/kubearmor-adapter)
+* [Trestle OSCAL Transformer](https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report/oscal-transformer)
+
+Since working groups are temporary and do not own code (see [governance](https://github.com/kubernetes/community/blob/master/committee-steering/governance/wg-governance.md)), we propose moving the Policy Reports API to `github.com/kubernetes-sigs/policy-report-api`.
+
+Promoting the Policy Report API will further increase adoption and signifies enhanced reliability, improved usability, and reinforces its significance within the cloud-native ecosystem.
+
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+The Policy Report API is already being used by several projects and tools in the Kubernetes community and CNCF ecosystem, as detailed above. Moving the API to a the `github.com/kubernetes-sigs/` GitHub org will provide a permanent home for the API, along with improved visibility and colloboration.
+
+### Goals
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+- Add `policy-report-api` as a new project under kubernetes-sigs i.e `github.com/kubernetes-sigs/policy-report-api`
+- Provide guidance on building consumers and producers
+
```

Comment:
Do we want to publish official artefacts for the API?
- YAML manifest?
- OCI image of Helm chart?
- something else?

Reply:
Here's what I suggest:
- Golang client set to reuse in producers and consumers
- Generated YAMLs
- API spec
- Docs